runtime/js-compute-runtime/builtins/request-response.cpp

```diff
@@ -1655,14 +1655,26 @@ JSObject *Request::create(JSContext *cx, JS::HandleObject requestInstance, JS::H
   JS::RootedValue fastly_val(cx);
   if (init_val.isObject()) {
     JS::RootedObject init(cx, init_val.toObjectOrNull());
+    JS::RootedValue overrideContentLength(cx);
     if (!JS_GetProperty(cx, init, "method", &method_val) ||
         !JS_GetProperty(cx, init, "headers", &headers_val) ||
         !JS_GetProperty(cx, init, "body", &body_val) ||
         !JS_GetProperty(cx, init, "backend", &backend_val) ||
         !JS_GetProperty(cx, init, "cacheOverride", &cache_override) ||
-        !JS_GetProperty(cx, init, "fastly", &fastly_val)) {
+        !JS_GetProperty(cx, init, "fastly", &fastly_val) ||
+        !JS_GetProperty(cx, init, "overrideContentLength", &overrideContentLength)) {
       return nullptr;
     }
+
+    if (overrideContentLength.isBoolean() && overrideContentLength.toBoolean()) {
```

Comment:
If we use `JS::ToBoolean`, then the value will be cast to a boolean, which is usually how people expect JavaScript APIs to work - for example [we use that here](https://github.com/fastly/js-compute-runtime/blob/4f399434c0959e902df03262dfceefdc16592afe/runtime/js-compute-runtime/builtins/request-response.cpp#L1963)